bgpd/bgp_updgrp.c

```diff
@@ -413,6 +412,56 @@ static unsigned int updgrp_hash_key_make(const void *p)
 		key = jhash_1word(jhash(peer->host, strlen(peer->host), SEED2),
 				  key);
 
+	if (bgp_debug_neighbor_events(peer)) {
+		zlog_debug(
+			"%pBP Update Group Hash: sort: %d UpdGrpFlags: %u UpdGrpAFFlags: %u",
+			peer, peer->sort, peer->flags & PEER_UPDGRP_FLAGS,
+			flags & PEER_UPDGRP_AF_FLAGS);
+		zlog_debug(
+			"%pBP Update Group Hash: addpath: %u UpdGrpCapFlag: %u UpdGrpCapAFFlag: %u route_adv: %u change local as: %u",
+			peer, (uint32_t)peer->addpath_type[afi][safi],
+			peer->cap & PEER_UPDGRP_CAP_FLAGS,
+			peer->af_cap[afi][safi] & PEER_UPDGRP_AF_CAP_FLAGS,
+			peer->v_routeadv, peer->change_local_as);
+		zlog_debug(
+			"%pBP Update Group Hash: max packet size: %u pmax_out: %u Peer Group: %s rmap out: %s",
```

Comment:
I think it's quite clear that it's `out`, we might drop the `out` keyword?

Reply:
I am not convinced it is clear :). That is why I put it there I have no problems with debug messages being wordy